docs/changes/units/17011.bugfix.rst

```diff
@@ -0,0 +1,6 @@
+Units involving special unicode symbols can now be parsed also if they are
+part of a composite unit, like "L☉/pc²".
+
+Similarly, CDS units ending in a 0 can now be part of composite units.
+
+Furthermore, a bug was fixed where "°C/s" was incorrectly interpreted as "°C/s^2".
```

Comment:
I think users would appreciate if the common part of these three cases is summarized in a single sentence and the details are in an explicit list:
```suggestion
The unit parsers are now much better at recognizing several unusual composite units:

  - units involving special unicode symbols, like "L☉/pc²",

  - units that include CDS units ending in a 0, like "eps0/s",

  - units including "°C". For example, "°C/s" is no longer incorrectly interpreted as "°C/s^2".
```

Reply:
Thanks, much better; used with slight modifications.